.github/workflows/tiledb-go.yml

```diff
@@ -7,6 +7,12 @@ on:
   pull_request:
     branches: [ master ]
 
+env:
+  # The version of TileDB to test against.
+  CORE_VERSION: "2.20.1"
```

Comment:
Why change the names?

Reply:
I find these names more descriptive than what I used before, and got inspired by Paul's unmerged PR https://github.com/TileDB-Inc/TileDB-Go/pull/231.